content/_data/changelogs/lts.yml

```diff
@@ -4785,6 +4785,346 @@
+
+  - type: rfe
+    category: rfe
+    pull: 5063
+    authors:
+    - fqueiruga
+    - Wadeck
+    message: |-
+      Reduce page load time by loading the administrative monitors popup on demand.
+      Allow keyboard navigation even when there are active administrative monitors.
+
+  - type: rfe
+    category: rfe
+    pull: 5176
+    authors:
+    - fqueiruga
+    - timja
+    message: |-
+      Use a more accessible color palette in configuration form tabs.
+
+  - type: rfe
+    category: rfe
+    pull: 5198
+    authors:
+    - res0nance
+    - alecharp
+    - batmat
+    references:
+      - pull: 5190
+      - pull: 5198
+      - issue: 64670
+    message: |-
+      Improve fingerprint save performance.
+
+  - type: rfe
+    category: rfe
+    pull: 5153
+    issue: 63222
+    authors:
+    - caseyduquettesc
+    message: |-
+      Add a system property <code>jenkins.agent.inboundUrl</code> to provide an alternate URL for inbound TCP agents.
+
+  - type: rfe
+    category: rfe
+    pull: 4962
+    authors:
+    - StefanSpieker
+    message: |-
+      Drop support for deprecated system properties: <code>hudson.model.Hudson.logStartupPerformance</code>, <code>hudson.model.Hudson.initLogLevel</code>, <code>hudson.model.Hudson.parallelLoad</code>, <code>hudson.model.Hudson.killAfterLoad</code> and <code>hudson.model.Hudson.workspaceDirName</code>.
+      Please use <code>jenkins.model.Jenkins.</code>-prefixed SystemProperties.
+
+  - type: rfe
+    category: rfe
+    pull: 5082
+    issue: 58101
+    authors:
+    - res0nance
+    message: |-
+      Reduce lock contention around Jenkins queue.
+
+  - type: rfe
+    category: rfe
+    pull: 5102
+    authors:
+    - daniel-beck
+    message: |-
+      Stop bundling CVS plugin.
+      Jenkins will no longer automatically install CVS plugin on startup if a plugin depending on Jenkins (then Hudson) 1.340 or earlier is discovered.
+      If you use a plugin that relies on the functionality provided by CVS plugin and manage plugins outside the Jenkins plugin manager, you will now need to ensure yourself that a recent release of CVS plugin is installed.
+      Jenkins will attempt to load such plugins but may fail at any time during startup or afterwards with <code>ClassNotFoundException</code> or similar.
+
+  - type: rfe
+    category: rfe
+    pull: 5244
+    authors:
+    - basil
+    references:
+      - pull: 5244
+      - url: "https://github.com/jenkinsci/ldap-plugin/releases/tag/ldap-2.3"
+        title: LDAP 2.3 changelog
+      - url: "https://github.com/jenkinsci/ldap-plugin/releases/tag/ldap-2.2"
+        title: LDAP 2.2 changelog
+      - url: "https://github.com/jenkinsci/ldap-plugin/releases/tag/ldap-2.1"
+        title: LDAP 2.1 changelog
+      - url: "https://github.com/jenkinsci/ldap-plugin/releases/tag/ldap-2.0"
+        title: LDAP 2.0 changelog
+    message: |-
+      Upgrade LDAP plugin from 1.26 minimum to 2.3.
+
```

Comment:
```suggestion
```

https://github.com/jenkinsci/jenkins/pull/5244#issuecomment-786034375: in weeklies, not LTS.

Reply:
Thanks for catching my mistake.  Reverted.  I will propose that change to the weekly changelogs so that it doesn't get lost in a future LTS

Reply:
Change proposed to https://github.com/jenkinsci/jenkins/pull/5244